src/components/SendTokensOne.js

```diff
@@ -192,7 +192,7 @@ class SendTokensOne extends React.Component {
 
     const renderSelectToken = () => {
       return (
-        <select value={this.state.selected.uid} className="ml-3" onChange={this.changeSelect}>
+        <select value={this.state.selected.uid} className="ml-3" title={hathorLib.wallet.isHardwareWallet() ? "This feature is disabled for hardware wallet" : "Select token"} disabled={hathorLib.wallet.isHardwareWallet() ? true : null} onChange={this.changeSelect}>
```

Comment:
Would you refactor and break it into multiple lines? I think it is easier to read and review.

Reply:
Done